_tools/release_docs/main.go

```diff
@@ -82,33 +131,50 @@ func processFile(prefix, filename string) error {
 var (
 	// Finds markdown links of the form [foo](bar "alt-text").
 	linkRE = regexp.MustCompile(`\[([^]]*)\]\(([^)]*)\)`)
+	// Finds markdown refence style link definitions
+	refRE = regexp.MustCompile(`(?m)^\[([^]]*)\]:\s+(.*)$`)
 	// Splits the link target into link target and alt-text.
 	altTextRE = regexp.MustCompile(`(.*)( ".*")`)
 )
 
-func rewriteLinks(fileBytes []byte) []byte {
-	return linkRE.ReplaceAllFunc(fileBytes, func(in []byte) (out []byte) {
-		match := linkRE.FindSubmatch(in)
-
-		visibleText := string(match[1])
-		linkText := string(match[2])
-		altText := ""
+func rewriteLinks(filename string, fileBytes []byte) []byte {
```

Comment:
This was mostly just refactored for reuse with `refRE`. We had some mesos documentation using reference style links that needed transforming.